src/building-blocks/FormItem.tsx

```diff
@@ -69,9 +69,31 @@ class FormItem extends Component<IFormFieldProps> {
     };
   }
 
+  private get getFieldConfig () {
```

Comment:
When we write getters, we remove the get from the name. So it would be `private get fieldConfig`